neon has version check for no apparent reason - drop it
--- neon-0.29.6/src/ne_socket.c~	2010-10-09 19:07:17.000000000 +0300
+++ neon-0.29.6/src/ne_socket.c	2015-01-10 11:47:34.384399098 +0200
@@ -1718,13 +1718,6 @@
 	return NE_SOCK_ERROR;
     }
 
-    /* If runtime library version differs from compile-time version
-     * number in major/minor/fix level, abort soon. */
-    if ((SSLeay() ^ OPENSSL_VERSION_NUMBER) & 0xFFFFF000) {
-        set_error(sock, _("SSL disabled due to library version mismatch"));
-        return NE_SOCK_ERROR;
-    }
-
     sock->ssl = ssl = SSL_new(ctx->ctx);
     if (!ssl) {
 	set_error(sock, _("Could not create SSL structure"));
